bpf/proxy/syncer.go

```diff
@@ -281,6 +346,18 @@ func (s *Syncer) startupSync(state DPSyncerState) error {
 	return nil
 }
 
+func (s *Syncer) startupSync(state DPSyncerState) error {
+	// try to build the previous maps based on the current state and what is in bpf maps.
+	// Once we have the previous map, we can apply the the current state as if we never
+	// crashed and apply only the diff using the regular code path.
```

Comment:
I think `crashed` is a bit misleading; we go through this path in a normal restart scenario too so I'd just say `restart` since that covers all cases.